x-pack/legacy/plugins/lens/public/indexpattern_plugin/filter_ratio.ts

```diff
@@ -64,10 +64,23 @@ export const calculateFilterRatio: ExpressionFunction<
       newRows.push(result);
     }
 
+    const newColumns = data.columns
+      .filter(col => !col.id.includes('filter-ratio'))
```

Comment:
I found this to be a magic string, and was confused. We're basically embedding metadata in our id and using that to categorize the column here. I think we should at least pull this out into a const so that if we change this in `to_expression`, it gets changed here, too.

Incidentally, this code looks wrong to me, but it's actually right. It looks like we're eliminating the filter-ratio column, then later trying to format that column. What we're actually doing is eliminating the extraneous column created by the filter ratio. Id is the count / whatever created by it. Not sure how we could clear this up, but wanted to point out the confusion.

Reply:
I see what you mean, but I think it's out of scope for this change. I added it to the "technical debt" list and will tackle it in a separate PR